utilities/console.py

```
@@ -54,18 +55,60 @@ def connect(self):
+
+                eof_index = len(patterns) - 2
+
+                if index_int == 0:
+                    LOGGER.info(f"{self.vm.name}: Sending username.")
+                    self.child.sendline(self.username)
+                elif index_int == 1:
+                    if self.password:
+                        LOGGER.info(f"{self.vm.name}: Sending password (masked).")
+                        self.child.sendline(self.password)
+                    else:
+                        raise ValueError("Password prompt received but no password provided.")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)